src/Microsoft.Identity.Web.OWIN/Microsoft.Identity.Web.OWIN.csproj

```diff
@@ -33,7 +33,7 @@
     <PackageReference Include="Microsoft.AspNet.Mvc" Version="5.2.7" />
     <PackageReference Include="Microsoft.AspNet.Web.Optimization" Version="1.1.3" />
     <PackageReference Include="Microsoft.AspNet.WebApi" Version="5.2.7" />
-    <PackageReference Include="Microsoft.IdentityModel.Protocols.WsFederation" Version="5.5.0" />
+    <PackageReference Include="Microsoft.IdentityModel.Protocols.WsFederation" Version="8.1.2" />
```

Comment:
Is this a breaking change if Owin customers were previously depending on `5.5.0`?

Reply:
I'll punt to @jmprieur I think we've been leaving it at this version intentionally, not remembering why though.

Reply:
Synced with JM offline, he agrees that we should not update this, and believes we don't actually use it.

Reply:
Thanks for updating us here!!!! It made it so easy for me to glance at this thread and ascertain what we decided on/what happened.